client/Pages/EventTypeCreate/Query.elm

```diff
@@ -103,7 +104,12 @@ viewQueryForm model =
 
               else
                 none
-            , if getValue FieldCleanupPolicy formModel.values == cleanupPolicies.delete then
+            , if
```

Comment:
@ferbncode just to make sure I understand the code, this is needed to change the behaviour of the view where it hides the field for retention time in case the choice is compact, right?

Reply:
@rcillo Right, the dropdown for selection of the retention time days is hidden when the policy is `compact`.